### PR TITLE
Testing - ensureArray create test

### LINKED ISSUE #8: Testing - ensureArray create test
Create test for file in src/ensureArray.ts

### PR DESCRIPTION
**Pull request by YPAzevedo**
Fixes #8

### CODE REVIEW

File: test/ensureArray.test.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { ensureArray } from '../src/ensureArray';
+
+it('any value passed should return array', () => {
+  const arrayWithTestSting = ensureArray(['test']);
+  const testString = ensureArray('test');
+
+  expect(arrayWithTestSting).toEqual(['test']);
+  expect(testString).toEqual(['test']);
+});
```
Comment (nobrayner) [inline test/ensureArray.test.ts]: You are testing two different cases here:
1. It should return an array with the given value
2. It return the given value if it is an array

I would advise separating these into two different tests - that way the intent is much clearer on what is happening :)

**Review by nobrayner — CHANGES_REQUESTED** [review]
Awesome! Thanks for getting this started! :smile:

File: test/ensureArray.test.ts (hunk 1 above)
Comment (YPAzevedo) [reply test/ensureArray.test.ts]: So it should the two `it`, right? also in the second one where i test for `It should return an array with the given value` should i add more types to test or only one thats different from an array, enough? 
```js
it('should return an array with the given value', () => {
  const testString = ensureArray('test');
  const testNumber = ensureArray(0);
  const testBoolean = ensureArray(true);

  expect(testString).toEqual(['test']);
  expect(testNumber).toEqual([0]);
  expect(testBoolean).toEqual([true]);
});
```
or only:
```js
it('should return an array with the given value', () => {
  const testString = ensureArray('test');

  expect(testString).toEqual(['test']);
});
```

Comment (nobrayner) [reply test/ensureArray.test.ts]: Hmm... Good question! Based on the fact that the `ensureArray` method is based on `any` type... I would check:
* A primitive
* A function
* An object

Between those three, I'd say you'd have it covered

Comment (YPAzevedo) [reply test/ensureArray.test.ts]: @nobrayner Yeah, makes sense. Updated the test add the assertions for those 3 different value types. let me know if need to change something else 👍

**Review by nobrayner — APPROVED** [review]
Now we just need to setup tests to run on CI :sweat_smile:

**Comment by YPAzevedo** [comment]
@nobrayner updated the test, added `toBe` on the first one since if it's an array it should be the same *value* it was passed in, so i added more test cases.

File: test/ensureArray.test.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  const getFourtyTwoArray = [(): number => 42];
+  const objFourtyTwoArray = [{ num: 42 }];
+
+  const ensureArrayFourtyTwoArray = ensureArray(fourtyTwoArray);
+  const ensureArrayGetFourtyTwoArray = ensureArray(getFourtyTwoArray);
+  const ensureArrayObjFourtyTwoArray = ensureArray(objFourtyTwoArray);
+
+  expect(ensureArrayFourtyTwoArray).toBe(fourtyTwoArray);
+  expect(ensureArrayGetFourtyTwoArray).toBe(getFourtyTwoArray);
+  expect(ensureArrayObjFourtyTwoArray).toBe(objFourtyTwoArray);
```
Comment (nobrayner) [inline test/ensureArray.test.ts:14]: Sweet! Good catch :+1:

**Review by nobrayner — APPROVED** [review]

**Comment by nobrayner** [comment]
Merging this, as fixing CI to run on PRs won't cause CI to run here.

**Pull request merged**